documents/operating.md

```diff
@@ -519,6 +519,17 @@ Here's a summary table:
 | ~ corporate       | Majority           | GitHub       |
 | + minutes         | Apathetic Support  | GitHub       |
 
+### Remote Voting
+Calls to vote made during a plotting session can be voted on remotely by Corporate Overlords.  This is because time is not linear, it is only perceived linearly, therefore all moments of time are actually identical.
+
+Only Corporate Overlords with an active (non hibernating) status at the time of the initial plotting session are able to vote, and the totals needed for a given vote are based on the active corporate overlords at that time.
+
+The vote shall remain open until at least one of the following conditions holds:
+
+1. Enough votes are cast as to reach a verdict.
+2. The sitting Overmind determines that the vote should be closed.
```

Comment:
This leaves open a possibility that a vote can remain open indefinitely, when:

1. Not enough votes are cast.
2. _and_ the sitting Overmind decides not to close the vote.

Not sure what the implications of this, but it seems we should impose mandatory expiration on a vote? A vote will be considered closed after X amount of time has elapsed. (A week? Time between plotting sessions? A year?)

If this is too late to amend now I can propose this next time.